src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -340,9 +340,7 @@ def _write_bdbag_samples_tsv(self, bundle_tsv: IO[str]) -> None:
             for bundle in doc['bundles']:
                 bundle_fqid = bundle['uuid'], bundle['version']
 
-                # Extract fields from the bundle
-                bundle_cells = {}
-                self._extract_fields([bundle], bundle_column_mapping, bundle_cells)
+                bundle_cells = {'entity:bundle_id': '.'.join(bundle_fqid)}
```

Comment:
This renders `bundle_column_mapping` an unused variable that should be eliminated.

Then again, I think we should also include bundle UUID and version as individual fields in which case we need the call to `_extract_fields`, passing `bundle_column_mapping`, as well as the custom 'entity:bundle_id' cell.